synapse/axon.py

```diff
@@ -451,6 +453,9 @@ def __init__(self, axondir, **opts):
 
         corepath = os.path.join(self.axondir, 'axon.db')
         self.core = s_cortex.openurl('sqlite:///%s' % corepath)
+        for name, model in axon_model.getBaseModels():
+            self.core.addDataModel(name, model)
```

Comment:
addDataModel is going to smash whatever the databmodel is into the cortex.  Which is okay for a new cortex; but if we have a model revision which requires code to run and actually make changes to existing tufos, those wont run.